core/model/src/sgx.rs

```diff
@@ -0,0 +1,29 @@
+pub use graphene::ias::AttestationResponse;
+use serde::{Deserialize, Serialize};
+use ya_service_bus::RpcMessage;
+
+/// Public SGX bus address.
+pub const BUS_ID: &str = "/public/sgx";
```

Comment:
Will these endpoints be called via network? If not, bindings could be local

Reply:
At first I thought it would be needed but in reality this will only be called locally. Changed.